helper/hashtable.c

```diff
@@ -60,6 +60,14 @@ typedef struct {
 	char name[ODPH_TABLE_NAME_LEN]; /**< table name */
 } odph_hash_table_imp;
 
+static uint32_t node_size(const odph_hash_table_imp *tbl)
+{
+	const uint32_t mask = sizeof(void *) - 1;
```

Comment:
This assumes that sizeof(void *) is a power of two, which is not guaranteed. Use of alignof instead of sizeof would be more accurate and would have the added benefit that all valid aligments are guaranteed to be powers of two.